x/merkledb/path.go

```diff
@@ -16,32 +18,10 @@ const (
 	Byte   byte = 8
 )
 
-type BranchFactor int
-
-const (
-	BranchFactorUnspecified BranchFactor = 0
```

Comment:
Removed this since I don't think we need this specific invalid branch factor anymore